### PR TITLE
Misc UI fixes for release 2.0.8

### LINKED ISSUE #5303: Grid icons are wrong
#### Summary
Grid icon artwork is wrong for many object types.
Screen Shot 2022-06-08 at 10 48 37 AM

#### Steps to Reproduce

1. Create a number of different object types, particularly Display Layout, Notebook, Example Imagery, Web Page, Gauge.
2. Note that the objects will have the correct icons in the tree view.
3. View the folder enclosing the objects in the main view in Grid view, and note that the icon artwork is wrong.

#### Environment

* Open MCT Version: 
* Deployment Type: 
* OS:
* Browser:

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [x] Does this impact a critical component?
- [x] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### LINKED ISSUE #5640: Misc UI problems
- [x] Make Display Layout frame edit handles overlap content, not resize it.
- [x] Fix search progress bar display so that progress bar doesn't lay under text.
- [x] Dial Gauges should center in their container, not left flush.
- [x] Add large icon for the aggregate telemetry type.
- [x] Add large icon for the Telemetry Table type. Closes #5303.
- [x] Fix descriptions for Telemetry Table, LAD Table and LAD Table Set.
- [x] Restore tree and Inspector pane's ability to collapse when dragged small enough.
- [x] Fix grippy icons in elements pool.
- [x] Fix value alignment in alphanumerics in Display Layouts.
- [x] Fix Locator in dialog to expand to use available space.
- [x] Fix styling of numeric inputs, especially in Firefox and on Linux.
- [x] Display Layout layout objects are erroneously using the unknown object icon because their type is not being properly defined and they end up using `UNKNOWN_TYPE` from TypeRegistry.js.

**Misaligned alphas in Display Layout**
image

**Numeric inputs in Firefox**
image

### PR DESCRIPTION
**Pull request by charlesh88**
Closes #5640
Closes #5303 

### Describe your changes:
- Better styling for search in progress and 'no results' messaging.
- Revised art and glyph for `icon-tabular-scrolling`.
- Change TelemetryTableType to use icon-tabular-scrolling icon and bg-icon.
- Fix bad/erroneous descriptions for Telemetry Table, LAD Table and LAD Table Set. Refined descriptions for other object types.
- Fixed bad value check that was causing panes not to collapse when dragged (by design) to a very narrow width.
- Visual style for grippy adjusted for better clarity on some monitors.
- Normalize proportional area used by alphanumeric's labels and values in Display Layouts.
- Fix margin strategy for object label in main view.
- Fixed Locator to dynamically size itself to use available vertical space.
- Added new `l-overlay-dialog` size parameter.
- Better approach to overlay sizing to make Properties dialogs larger vertically and fullscreen overlays more apparent to the user.
- Normalized approach to numeric inputs to fix cross-browser inconsistency with input spinners.
- Fixed layout problem in `Datetime.vue` component due to numeric input spinners changes.
- Added type definitions for Display Layout drawing objects; code modded in `ObjectName.vue` to grab layout object `cssClass` from layoutItem.type. Big thanks to @jvigliotta for the assist with this!

### All Submissions:
* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist
* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Review by cr4yfish — CHANGES_REQUESTED** [review]
Glanced over it and looks good. 
But I think you forgot to take out a console log. Tagged it.

File: src/plugins/displayLayout/plugin.js [hunk 1]
```diff
@@ -125,6 +126,12 @@ export default function DisplayLayoutPlugin(options) {
                 return true;
             }
         });
+
+        for (const [type, definition] of Object.entries(DisplayLayoutDrawingObjectTypes)) {
+            openmct.types.addType(type, definition);
+            console.log(type, definition);
```
[the remark this answers is not in the document]
Comment (charlesh88) [reply src/plugins/displayLayout/plugin.js:132]: Thanks!

**Review by michaelrogers — APPROVED** [review]
These changes look good to me.

File: src/plugins/displayLayout/plugin.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (akhenry) [reply src/plugins/displayLayout/plugin.js]: @cr4yfish Thanks for your help reviewing this!

**Review by akhenry — CHANGES_REQUESTED** [review]
Looks like there are changes from Master in here. Blocking until we can clean it up.

**Comment by ozyx** [comment]
Retargeted against master. This branch will need to be cherry-picked to get into a good state.

**Comment by charlesh88** [comment]
I will redo this against release/2.0.8.

**Pull request closed without merging**